### PR TITLE
Change interface

### PR DESCRIPTION
**Pull request by diegocosta-dev**

### CODE REVIEW

File: src/utils/calcOperations.ts [hunk 1]
```diff
@@ -13,8 +17,42 @@ export const getCalcOperationsByType = (operations: Operation[], type: Transacti
...
+
+export const getTotalCreditCardExpenses = (operations: Operation[]) => {
+  const operationFiltered = operations.filter(
+    (operation) => operation.type === TransactionType.CreditCard,
+  );
+  const total = operationFiltered.reduce(sumOperation, 0);
+  return total;
+};
+
+export const PlannedBalance = (operations: Operation[]) => {
```
Comment (davidcostadev) [inline src/utils/calcOperations.ts:52]: The recommendation is to use the first letter as uppercase only for classes or react components

File: src/pages/Dashboard/NavBar/MonthNavigation.tsx [hunk 2]
```diff
@@ -30,10 +33,19 @@ const MonthNavigation = () => {
       <ArrowButton isLoading={loading} onClick={goToPreviewMonth}>
         <ArrowIosBack />
       </ArrowButton>
+      <div
```
Comment (davidcostadev) [inline src/pages/Dashboard/NavBar/MonthNavigation.tsx:36]: Please use styled components instead

Comment (davidcostadev) [reply src/pages/Dashboard/NavBar/MonthNavigation.tsx:36]: You could use styled(Text)

**Review by davidcostadev — CHANGES_REQUESTED** [review]

File: src/hooks/useSidebarCollapse.tsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ const isSidebarOpenContext = createContext<SidebarContext>({
 });
 
 export default function SidebarProvider({ children }: Props) {
+  !localStorage.getItem('sidebarCollapse') && localStorage.setItem('sidebarCollapse', 'true');
   const [isSidebarOpen, setSidebarVisibility] = useState(
     localStorage.getItem('sidebarCollapse') === 'true',
```
Comment (davidcostadev) [inline src/hooks/useSidebarCollapse.tsx:19]: What about to invert the condition:
`localStorage.getItem('isSidebarClose') === 'true'`

**Review by davidcostadev — CHANGES_REQUESTED** [review]

File: src/hooks/useSidebarCollapse.tsx (hunk 3 above)
Comment (diegocosta-dev) [reply src/hooks/useSidebarCollapse.tsx:19]: when we login for the first time the sidebarCollapse is undefined, if sidebarCollapse  undefined set sidebarCollapse  to "true"

**Review by davidcostadev — APPROVED** [review]

**Pull request merged**